README.md

```diff
@@ -1,32 +1,88 @@
-# Dev Container Templates: Self Authoring Guide
+# DevContainer Templates
+A collection of LocalStack related and managed DevContainer Templates.
 
-> This repo provides a starting point and example for creating your own custom [Dev Container Templates](https://containers.dev/implementors/templates), hosted for free on GitHub Container Registry.  The example in this repository follows the [Dev Container Template distribution specification](https://containers.dev/implementors/templates-distribution/).  
+>Development Container Templates are source files packaged together that encode configuration for a complete development environment.
+A Template can be used in a new or existing project, and a [supporting tool](https://containers.dev/supporting) will use the configuration from the Template to build a development container.
```

Comment:
"Template" is capitalized here, but not in subsequent usages. Is this on purpose? I think using a common spelling, even though it is a devcontainer feature as well, would make sense for readability.

Reply:
Fair point, I run through the docs again to see where we're talking about DevContainer Templates as Templates and just templates in general.